batch7crud-roll5/src/main/java/com/lftechnology/batch7crud/services/EmployeeServiceImpl.java

```diff
@@ -16,12 +20,16 @@ public Employee fetchById(int id) throws DataException {
         return employeeDAO.fetchById(id);
     }
 
-    public void create(Employee employee) throws DataException {
-        employeeDAO.create(employee);
+    public void create(Employee employee) throws DataException, ValidationExceptions {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: com.lftechnology.batch7crud.exception.DataException, !unknownSymbol! [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1160)